libs/sdk-ui-theme-provider/src/derivedColors.ts

```diff
@@ -0,0 +1,256 @@
+// (C) 2021 GoodData Corporation
+import { transparentize, darken, lighten, mix, setLightness } from "polished";
+import { IThemePalette, IThemeChart, ThemeColor } from "@gooddata/sdk-backend-spi";
+import { CssProperty, getCssProperty } from "./cssProperty";
+
+// keep it in sync with SCSS:$gd-color-text-light
+const GD_COLOR_TEXT_LIGHT = "#fff";
+const GD_COLOR_TEXT = "#464e56";
```

Comment:
Are we always mixing with white? I thought that these two colors would also change based on dark mode..

But I guess mixing with black is nonsense?

Reply:
Nevermind, I just noticed, we don't mix. We just use it as fallback..